packages/@okta/vuepress-site/docs/release-notes/index.md

```diff
@@ -15,3 +15,12 @@ To verify the current release for an org, check the footer of the administrator
 ![Release Number in Footer](/img/release_notes/version_footer.png)
 
 > **Note:** Changes to Okta unrelated to API Products are published in the [Okta Release Notes](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_okta_relnotes).
+
+
+<div class="next-section">
+  <RouterLink to='/docs/release-notes/2021'>
+    <button class="button is-button-cerise is-button-small" routerLink="/docs/release-notes/2021">
```

Comment:
Is there a reason we set the `to` and `routerLink` on both components?

Reply:
thanks for letting me know, it's was forgotten for removing